proto/decentraland/kernel/comms/rfc4/comms.proto

```diff
@@ -101,6 +101,7 @@ message ProfileResponse {
 message Chat {
   string message = 1;
   double timestamp = 2;
+  optional string from = 3;
```

Comment:
Hi! Could you add a comment to explain what this is? Without reading the chat, it’s not entirely clear to me.

Also, what are the use cases? Is it just for the community, or does it also include private chats?

Isn’t an enum for message type enough?

And finally. Just a suggestion. What about using “channel” or “topic” instead of “from”? I think that would be more descriptive.

Reply:
Thanks for your suggestions regarding the topics. We are currently implementing a new mechanism using our SFU to address LiveKit topics being broadcasted to everyone connected to the room. The SFU will now be responsible for forwarding messages only to the relevant community members in community chats.

As for the reasoning behind using `channel` or `topic`, the goal is to include the identity of the original sender in the message. That's why I proposed adding a `from` property.